src/routes/resources/+page.svelte

```diff
@@ -55,21 +82,29 @@
       }
     }
 
+    // search to rely on displayedResources
+    filterByTags = displayedResources;
+
     // Force svelte re-render
     displayedResources = displayedResources;
   }
 
+  // Update the displayedResourceLimit based on the scroll position
   const updateLimit = (event: CustomEvent<{displayLimit: number}>) => {
     const {displayLimit} = event.detail
     displayedResourceLimit = displayLimit
   }
+
+  let mainH1El: HTMLHeadingElement | null;
```

Comment:
some of these look like artifacts possibly from merge conflicts; is this in use in the file?

Reply:
Correct, this line of code is not in use and can be removed.